module/data/actors/actor-crew.mjs

```diff
@@ -31,4 +31,14 @@ export default class cgdCrew extends cgdActorBase {
     if (bird.length == 1)
       this.parent.updateSource({ ["system.bird"]: bird[0].uuid });
   }
+
+  recalculateSupplies() {
+    const crewExplorers = Object.values(this.roles);
```

Comment:
To avoid issues, it would be better to remove the empty/null entries from the list.
```suggestion
    const crewExplorers = Object.values(this.roles).filter(x => x);
```

Reply:
Definitely, addressed in [acb6825](https://github.com/hodpub/coriolis-tgd/pull/41/commits/acb68257346ac8e2c7925130f89f6e778f8d5a7a)